skimage/color/colorconv.py

```diff
@@ -844,10 +844,10 @@ def gray2rgb(image, alpha=None):
             is_rgb = True
 
     if is_rgb:
-        if alpha == False:
+        if alpha is False:
             image = image[..., :3]
 
-        elif alpha == True and not is_alpha:
+        elif alpha is True and is_alpha is False:
```

Comment:
Just tried with mine `flake8`: it is OK with both versions.
I do not insist on changing your fix. :)